packages/back-end/src/api/openapi/payload-schemas/PostExperimentPayload.yaml

```diff
@@ -41,6 +41,15 @@ properties:
     type: array
     items:
       type: string
+  activationMetric:
+    type: string
+    description: Users must convert on this metric before being included
+  segmentId:
```

Comment:
We don't use `id` in the name of any other field (e.g. `activationMetricId`). Can we just use the same naming and call it `ID` in the description?

Reply:
it's already `segmentId` in the response object (settings). I don't want to deviate from that

Reply:
Yep, that takes precedence. I didn't notice that, ty.